src/home.jsx

```diff
@@ -0,0 +1,9 @@
+const Home = () => {
```

Comment:
To maintain a consistent folder structure, please move the homepage file in a `/pages` directory.